src/main/java/uk/gov/hmcts/darts/retention/exception/RetentionApiError.java

```diff
@@ -47,6 +47,34 @@ public enum RetentionApiError implements DartsApiError {
         RetentionErrorCode.RETENTION_POLICY_TYPE_ID_NOT_FOUND.getValue(),
         HttpStatus.NOT_FOUND,
         RetentionTitleErrors.RETENTION_POLICY_TYPE_ID_NOT_FOUND.toString()
+    ), NON_UNIQUE_POLICY_NAME(
```

Comment:
general comment - we have a lot of this horrible looking boilerplate code - just wondered (not in scope for this ticket) if we could/should do this a better way?

Reply:
Yeah it's tricky. This is actually an improvement on how it used to be - we used to hardcode the error codes and the titles directly into the enum here. We now define that info in the openapi (with the aim of self-documenting the error details), and use this enum to tie everything together.

Definitely open to suggestions on how to make this better though, if you have any ideas!